File: wireless/sims/instance-post-example-1/instance-post-example-1.5.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account SID and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'

@client = Twilio::REST::Client.new(account_sid, auth_token)

sim = @client.preview.wireless
  .sims("DEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA")
  .update(
    status: 'active',
    callbackUrl: 'https://sim-manager.mycompany.com/sim-update-callback/AliceSmithSmartMeter'
    callbackMethod: 'POST'
  )

puts sim

File: wireless/sims/instance-post-example-1/instance-post-example-1.5.x.php
<?php
// Get the PHP helper library from twilio.com/docs/php/install
require_once '/path/to/vendor/autoload.php'; // Loads the library
use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token = "your_auth_token";
$client = new Client($sid, $token);

$sim = $client->preview->wireless
    ->sims("DEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA")
    ->update([
        'Status' => 'active',
        'CallbackUrl' => 'https://sim-manager.mycompany.com/sim-update-callback/AliceSmithSmartMeter',
        'CallbackMethod' => 'POST'
    ]);

print_r($sim);
